fic_modules/git.py

```diff
@@ -244,15 +260,11 @@ def filter_git_tag(repository_name, repository_team, repository_path):
      github.GithubException.GithubException: 502 {'message': 'Server Error'}
     """
     number = 0
-    version_path = REPOSITORIES\
-        .get("Github")\
-        .get(repository_name)\
-        .get("configuration")\
-        .get("version-path")
+
     latest_releases = get_version(repository_name, repository_team)
     if get_version_from_json(repository_name) == \
             latest_releases.get("latest_release").get("version"):
-            logger.info("No new changes entered production")
+        LOGGER.info("No new changes entered production")
```

Comment:
Lost indentation.
Was this on purpose?

Reply:
I think it was, AFAICS the code from the line 266 looks to be a continuation from the line 265 and the 267 was double tabbed to match the continuation from the line 266 which is not correct.